src/tests/resources/csharp-missing-assembly-name.csproj

```diff
@@ -5,7 +5,7 @@
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>{6BF38215-C9D9-418D-9D81-4DECBA679223}</ProjectGuid>
-    <OutputType>Library</OutputType>
+    <OutputType>Exe</OutputType>
```

Comment:
Like Manfred, I don't understand why you made this change, could you explain please?

Reply:
The original use of this file was to make sure the extension threw when AssemblyName was missing. I repurposed it to 1) use a default AssemblyName but also to 2) use a non-default OutputType to ensure that both default values weren't being returned, as a bit of black box verification. I can certainly change it back if that's a bit obtuse, but it heads off a potential incorrect code path.